http/identity-tenant-regression-test.http

```diff
@@ -0,0 +1,214 @@
+### Create Tenant
+# @name create-tenant
+POST {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/tenants
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "tenantId": "testTenant1",
+  "name": "TestTenant1",
+  "description": "A test tenant."
+}
+
+> {%
+  client.test("Tenant creation should be successful", function () {
+    client.assert(response.status === 201, "Response status is not 201");
+    client.global.set("TENANT_ID", response.body.tenantId);
+  });
+%}
+
+### Get Tenant
+# @name get-tenant
+< {%
+  import {wait} from "js/wait";
+  wait(3);
+%}
+GET {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/tenants/{{TENANT_ID}}
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Accept: application/json
+
+> {%
+  client.test("Tenant retrieval should be successful", function () {
+    client.assert(response.status === 200, "Response status is not 200");
```

Comment:
🔧 Should we assert that the `tenantId` is the correct one here?

I'm not familiar with this type of test, so I assume there is a "test context", and the `TENANT_ID` is already known from [here](https://github.com/camunda/camunda/pull/29402/files#diff-c06fc5b6d5241e2fc6f5c1690d47d9d434c959295c2da95d60e7b73608d4377eR17).